go.mod

```diff
@@ -1,7 +1,7 @@
 module github.com/kubernetes-sigs/aws-fsx-csi-driver
 
 require (
-	github.com/aws/aws-sdk-go v1.29.9
+	github.com/aws/aws-sdk-go v1.35.6
```

Comment:
try also to run 'go mod tidy', i think go.sum should have been updated too

Reply:
Ran go mod tidy and updated the pr with go.sum. I also verified go make still works.